spec/models/claim/pii_scrubber_spec.rb

```diff
@@ -0,0 +1,102 @@
+require "rails_helper"
+
+RSpec.describe Claim::PiiScrubber, type: :model do
+  let(:over_two_months_ago) { 2.months.ago - 1.day }
+
+  it "does not delete details from a submitted claim" do
+    freeze_time do
+      claim = create(:claim, :submitted, updated_at: over_two_months_ago)
+
+      Claim::PiiScrubber.new.scrub_completed_claims
+      cleaned_claim = Claim.find(claim.id)
+
+      expect(cleaned_claim.attributes).to eq(claim.attributes)
+    end
+  end
+
+  it "does not delete details from an approved but unpaid claim" do
+    freeze_time do
+      claim = create(:claim, :approved, updated_at: over_two_months_ago)
+
+      Claim::PiiScrubber.new.scrub_completed_claims
+      cleaned_claim = Claim.find(claim.id)
+
+      expect(cleaned_claim.attributes).to eq(claim.attributes)
+    end
+  end
+
+  it "does not delete details from a newly rejected claim" do
+    freeze_time do
+      claim = create(:claim, :rejected)
+
+      Claim::PiiScrubber.new.scrub_completed_claims
+      cleaned_claim = Claim.find(claim.id)
+
+      expect(cleaned_claim.attributes).to eq(claim.attributes)
+    end
+  end
+
+  it "does not delete details from a newly paid claim" do
+    freeze_time do
+      claim = create(:claim, :approved)
+      create(:payment, :with_figures, claims: [claim])
+
+      Claim::PiiScrubber.new.scrub_completed_claims
+      cleaned_claim = Claim.find(claim.id)
+
+      expect(cleaned_claim.attributes).to eq(claim.attributes)
+    end
+  end
+
+  it "deletes details from an old rejected claim" do
+    claim = create(:claim, :rejected, updated_at: over_two_months_ago)
+
+    Claim::PiiScrubber.new.scrub_completed_claims
+    cleaned_claim = Claim.find(claim.id)
+
+    expect(cleaned_claim.attributes).to_not eq(claim.attributes)
```

Comment:
This feels like a pretty weak assertion as any number of things might cause the attributes to be different. Instead I think I'd prefer to see the assertions from the `"deletes the fields we expect"` and the one on the timestamp in the bottom spec duplicated in this spec and the one for old paid claims so we end up with two combined `it "scrubs the PII fields on BLAH claims, setting a pii_removed_at timstamp"` examples that define what happens to specific types of claims in their entirety. It creates some duplication, but a little duplication is acceptable in specs in the service of making the specs more robust and easier to follow.